FitNesseRoot/DbFit/AcceptanceTests/JavaTests/OracleTests/LibraryTests/StoredProcWithException/content.txt

```diff
@@ -1,12 +1,12 @@
-!3 Use ''Execute Procedure With Exception'' to check if a stored proc will throw an exception
+!3 Use ''Execute Procedure Expect Exception'' to check if a stored proc will throw an exception
 
 !|Execute|!-
 create or replace procedure TestProc1(name varchar2, strlength out number) as
 begin
-        if (name='xx') then
-          raise_application_error(-20001,'test exception');
-        end if;
-	strlength:=length(name);
+    if (name='xx') then
```

Comment:
Now applied.

Perhaps we should adopt some standard SQL formatting via some free web service such as http://www.dpriver.com/pp/sqlformat.htm, but I don't know how we could define a standard config of our chosing. Perhaps use the service's default. Or maybe there are other better free desktop tools.